Lesson26/Kadir/script.js

```diff
@@ -17,10 +17,20 @@ function fetchPosts() {
 				const editButton = document.createElement('a');
 				editButton.classList.add('button');
 				editButton.classList.add('button--danger');
+				editButton.classList.add('button--edit');
 				editButton.textContent = "Edit";
 				editButton.href = `./edit-post/edit-post.html?postId=${post.id}`;
 				listItem.appendChild(editButton);
 
+				const deleteButton = document.createElement('a');
+				deleteButton.classList.add('button');
+				deleteButton.classList.add('button--delete');
+				deleteButton.textContent = "Delete";
+				deleteButton.id="delete-post";
+				deleteButton.href = `${URL}/${post.id}`;
```

Comment:
The behaviour of this button is a bit weird, it redirects user to another website instead of sending HTTP request to delete a post. Remove the href from the button and instead add a JS function that would be triggered when the button is clicked. That function should send the delete request the the API.